chain/client/src/stateless_validation/chunk_endorsement/mod.rs

```diff
@@ -28,7 +31,9 @@ impl ChunkEndorsementsState {
 
 /// Module to track chunk endorsements received from chunk validators.
 pub struct ChunkEndorsementTracker {
+    epoch_manager: Arc<dyn EpochManagerAdapter>,
     pub tracker_v1: tracker_v1::ChunkEndorsementTracker,
+    pub tracker_v2: tracker_v2::ChunkEndorsementTracker,
```

Comment:
do we expect both of the these initialized at the same time? otherwise could these be put in an enum?

Reply:
Yes, this is for the transition period between the two protocol versions. Once we shift to the new protocol version, we can get rid of tracker_v1